nss/src/coverage.go

```diff
@@ -0,0 +1,4 @@
+// Package coveragesrc file is only here so that it’s recognized as a go package when computing coverage
+package coveragesrc
+
+import "C"
```

Comment:
There's a .c file in the src/ directory and Go was complaining about it, so I had to add this `import "C"` clausure to make it stop.

Reply:
This is not a lint problem, I think. I get an error saying: 
> C source files not allowed when not using cgo or SWIG: db_override.c `go list`